client/public/index.html

```diff
@@ -11,6 +11,10 @@
     />
     <link rel="apple-touch-icon" href="%PUBLIC_URL%/logo192.png" />
     <link rel="manifest" href="%PUBLIC_URL%/manifest.json" />
+    <link href="//netdna.bootstrapcdn.com/bootstrap/3.0.0/css/bootstrap.min.css" rel="stylesheet" id="bootstrap-css">
```

Comment:
It was great you added bootstrap. It updated the fonts and improved certains visuals but I belive we are not using bootstrap in this project . So as a client side project we should try not to increase the load time or bundle size

Reply:
Its a nice thought not to increase bundle size so any suggestion on beautifying the page without bootstrap

Reply:
I think since since we are using mui there must be a theme.js in the application for styling the whole application you can use that one

Reply:
Fine I'll try that